futures-util/src/compat/mod.rs

```diff
@@ -6,7 +6,7 @@ mod executor;
 pub use self::executor::{Executor01CompatExt, Executor01Future, Executor01As03};
 
 mod compat01as03;
-pub use self::compat01as03::{Compat01As03, Future01CompatExt, Stream01CompatExt};
+pub use self::compat01as03::{Compat01As03, Future01CompatExt, Stream01CompatExt, Sink01CompatExt};
```

Comment:
I'm not 100% sure, but I guess `Compat01As03Sink` should be included here, right?

Reply:
Yes that is probably a good idea. Will push that soon.

Reply:
Sorry to be a pain, but `Compat01As03Sink` has also been included here: https://github.com/rust-lang-nursery/futures-rs/blob/master/futures/src/lib.rs#L90 Anyway, thank for your great work!

Reply:
No worries! Thanks for pointing these out 😄 , just pushed the change.